### DISCUSSION TITLE
不会安装怎么办啊，QA里面都看了没搞定。老哥能否指导一下，谢谢啦

### DISCUSSION THREAD

**Discussion by altmanPowell** [Vespa314/chan.py#63] [Q&A]
我python环境啥的都安装好了，但是项目好像配置和启动项呢，老哥能否指导一下怎样本地安装部署，谢谢啦

**Comment by Vespa314** [comment]
看quickguide.md

这是一个计算框架，需要自己开发策略对接的，不是一个开箱即用的app。

**Comment by altmanPowell** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
